BUILDING.txt

```diff
@@ -326,17 +326,12 @@ to update SNAPSHOTs from external repos.
 ----------------------------------------------------------------------------------
 Importing projects to eclipse
 
-When you import the project to eclipse, install hadoop-maven-plugins at first.
+At first, install program files at the top of the source tree.
```

Comment:
@rinikeda How about keeping mention about hadoop-maven-plugins like `s/program files/artifacts including hadoop-maven-plugins/`?

Reply:
I think that is better expression. Thank you.